src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const books = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addNewBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => ({
+      ...state,
+      books: state.books.filter((book) => book.id !== action.payload),
+    }),
+  },
+});
+
+export const { addNewBook, removeBook } = books.actions;
```

Comment:
- [ ] Kindly be noticed that there should be an `ADD_BOOK` and `REMOVE_BOOK` as action types, The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`.
- [ ] Also you should create and exports **Action Creators**. (`addBook` and `removeBook` functions)